packages/console/src/pages/Organizations/index.tsx

```diff
@@ -29,10 +34,18 @@ type Props = {
 
 function Organizations({ tab }: Props) {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+  const { currentTenantId, isDevTenant } = useContext(TenantsContext);
+  const { data: currentPlan } = useSubscriptionPlan(currentTenantId);
+
   const { navigate } = useTenantPathname();
   const [isCreating, setIsCreating] = useState(false);
   const { configs, isLoading: isLoadingConfigs } = useConfigs();
   const isInitialSetup = !isLoadingConfigs && !configs?.organizationCreated;
+  const isFreePlan = currentPlan?.name === ReservedPlanName.Free;
```

Comment:
per usage, better to be `isOrganizationsDisabled` or something

Reply:
Updated

Reply:
Also, should avoid using `ReservedPlanName` since it's only used in the frontend for displaying purpose, use `ReservedPlanId` instead if you need.